docs/pages/guides/using-sentry.md

```diff
@@ -65,12 +67,25 @@ Sentry.init({
   enableInExpoDevelopment: true,
   debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
 });
+```
+
+#### Usage
+
+Depending on which platform you are on (mobile or web), use the following functions in order to capture/catch exceptions:
```

Comment:
```suggestion
Depending on which platform you are on (mobile or web), use the following functions to access any `@sentry/*` exports for instrumentation, performance, and so on:

- For React Native, access any `@sentry/react-native` exports with `Sentry.Native.*`
- For web, access any `@sentry/browser` exports with `Sentry.Browser.*`
```

As per Kim's suggestion, let's be more verbose about accusing any exports from `@sentry` packages.